### ISSUE TITLE
Bug: new version notification

### ISSUE THREAD

**Issue by jordanh** [ParabolInc/action#1764]
## Issue - Bug

Once upon a time, our software would pop a toast when a new version had been pushed to the server. Now, I see the socket disconnect message but no toast on new version.

Perhaps we regressed?

Granted, I know we want to eliminate the need to logout all together. But, let's fix this on the way if we can.

- **OS:** macOS
- **Browser:** Chrome`
- **Estimated effort:** 11 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### Acceptance Criteria (optional)
When a new software version is pushed to the server we
   - pop a toast indicating it's time to log out
   - when clicked we log the user out, download the new assets, allow them to log in again

**Comment by mattkrick** [comment]
the logic is still there & is working in development (heck, this is the entire reason for the PR to apollo).

the chain of events is:
- server goes down
- socket disconnected toast
- server goes up
- upgrade toast
- socket reconnected toast

alternatively, if you were on the landing page & didn't establish a socket connection yet:
- server goes down
- server goes up
- you log in
- socket connects
- upgrade toast

basically, any time a socket connects or reconnects, it checks for the version. any steps to reproduce the bug?

**Comment by tayamueller** [comment]
closed in favor of #1786
